di_website/blog/models.py

```diff
@@ -100,7 +117,10 @@ class BlogArticlePage(TypesetBodyFootnoteMixin, HeroMixin, Page):
         help_text='This date will be used for display and ordering',
     )
 
+    colour = models.CharField(max_length=256, choices=COLOUR_CHOICES, default=RED)
+
     content_panels = Page.content_panels + [
+        FieldPanel('colour'),
```

Comment:
What's this supposed to do? As far as I can tell, all it does is change the colour of the infographic share button ... which doesn't make it very useful. It was for publications which had pages in all kinds of colours, but blogs are always in the classic red.